CONTRIBUTING.md

```diff
@@ -1,8 +1,22 @@
-# Contributing
+# Contributing Guidelines
 
-Unfortunately, this repository is not yet enabled for pull request contributions.
-However, you are encouraged to [open an issue](https://github.com/IBM-Cloud/cloud-provider-ibm/issues)
-to request new features or to get bugs triaged and fixed. Thank you.
+The **cloud-provider-ibm** project accepts contribution via github [pull request](https://help.github.com/articles/about-pull-requests/). 
+This document outlines the process to help get your contribution accepted. Please also read the [Kubernetes contributor guide](https://github.com/kubernetes/community/blob/master/contributors/guide/README.md).
+
+## Sign the Contributor License Agreement
```

Comment:
This may not be required now, But once we eventually move to under K8s we need to have it.